src/thing_data.c

```diff
@@ -226,6 +226,9 @@ TbBool thing_exists(const struct Thing *thing)
     return true;
 }
 
+/**
+ * @param In a player hand, excludes creature controlled limbo like hero gates
+  */
 TbBool thing_is_in_limbo(const struct Thing* thing)
```

Comment:
what is a limbo?

Reply:
Limbo is 'in between places', being halfway between existing and not existing.
So, creatures that flee through a hero gate, will be in limbo, because they are not on the map, but the thing still exists. Same is creatures in your hand, they are stored at the location you pick them up, but are not really there.

This PR is fixing a bug, where you can win maps by chasing the knight through the hero gate with cave-in. He would return after a few seconds, but in the meantime it would have counted as a win.